### PR TITLE
Prepare to publish draft-ietf-ppm-dap-06

### PR DESCRIPTION
**Pull request by tgeoghegan**
- Bump version tags in domain separation strings
 - Update VDAF reference
 - Update changelog

### CODE REVIEW

File: draft-ietf-ppm-dap.md [hunk 1]
```diff
@@ -114,6 +114,14 @@ seen in the clear by any server.
 
 (\*) Indicates a change that breaks wire compatibility with the previous draft.
 
+06:
+
+- Bump draft-irtf-cfrg-vdaf-06 to 07 {{!VDAF}}. (\*)
+
+- Adopt revised ping-pong interface in draft-irtf-cfrg-vdaf-07 (#494).
+
+- Bump version tag from "dap-05" to "dap-06". (\*)
```
Comment (tgeoghegan) [inline draft-ietf-ppm-dap.md:123]: We might also take #495, in which case we should add an item here.

**Comment by wangshan** [comment]
just pointing out the title of this PR should be ppm-dap-06

**Comment by tgeoghegan** [comment]
> just pointing out the title of this PR should be ppm-dap-06

Thank you, you are correct. I got mixed up with VDAF-07. I fixed the PR title and the commit.

**Comment by tgeoghegan** [comment]
I'm waiting for draft-irtf-cfrg-vdaf-07 to be published so we can verify that this document's references to that revision are OK, and then I will merge this and cut draft-ietf-ppm-dap-06.

File: draft-ietf-ppm-dap.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ venue:
   github: "ietf-wg-ppm/draft-ietf-ppm-dap"
   latest: "https://ietf-wg-ppm.github.io/draft-ietf-ppm-dap/draft-ietf-ppm-dap.html"
 
-v: 4
+v: 3
```
Comment (cjpatton) [inline draft-ietf-ppm-dap.md:15]: nit: It may be worth noting this change in the commit message.

**Review by cjpatton — APPROVED** [review]

**Pull request merged**